va/validation-authority.go

```diff
@@ -387,8 +387,9 @@ func (va *ValidationAuthorityImpl) validateTLSWithZName(identifier core.AcmeIden
 	}
 
 	challenge.Error = &probs.ProblemDetails{
-		Type:   probs.UnauthorizedProblem,
-		Detail: "Correct zName not found for TLS SNI challenge",
+		Type: probs.UnauthorizedProblem,
+		Detail: fmt.Sprintf("Correct zName not found for TLS SNI challenge. Found %s",
```

Comment:
I just looked at this again and it should be `%#v` to prevent folks getting blanks here, no?